Features/Swap/Sources/ViewModels/SwapSceneViewModel.swift

```diff
@@ -88,6 +94,14 @@ public final class SwapSceneViewModel {
 
     var providerField: String { Localized.Common.provider }
 
+    var swapEstimationTitle: String { Localized.Swap.EstimatedTime.title }
+    var swapEstimation: String? {
```

Comment:
```suggestion
    var swapEstimationText: String? {
```

can you add unit test for it?

Reply:
@gemcoder21 Let me create an issue for SwapSceneViewModel? There's a lot we can cover in tests